npm/vite-dev-server/src/plugins/cypress.ts

```diff
@@ -76,7 +76,7 @@ export const Cypress = (
       indexHtmlContent = indexHtmlContent.replace(
         '<head>',
         `<head>
-        ${scriptTagsToInject.map((script) => script.toString())}
+        ${scriptTagsToInject.map((script) => script.toString()).join('')}
```

Comment:
Is this essential or just nice to have? Just curious

Reply:
Check out the before video (near the end), it was actually rendering a comma that was visible. I don't think Percy would have caught this since we don't snapshot our system tests. Going to look into this a bit more, since like you said I shouldn't have seen the comma if it was in the `<head>` so somehow it's making it's way to the `body`

Reply:
After digging into this, I discovered that if we insert text into the head that doesn't parse correctly (in this case, we had the comma due to the stringified array), it will get shifted into the body.
Before:
![Screen Shot 2022-08-02 at 12 13 04 PM](https://user-images.githubusercontent.com/25158820/182434463-3310b173-1ca7-4ef1-8c2b-29e233e30963.png)

After:
![Screen Shot 2022-08-02 at 12 12 25 PM](https://user-images.githubusercontent.com/25158820/182434492-d175ecdf-aca6-4541-b487-52b89b6724c1.png)

So this fix is more important that I thought.

Reply:
Why do browsers do things like "move text from `<head>` into `<body>`???? :-1:

Reply:
Does not seem to impact percy what-so-ever though! Weird.